File: src/main/java/com/david/knowledgegraph/KnowledgeGraph.java
package com.david.knowledgegraph;

import java.io.File;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.Paths;


public class KnowledgeGraph
{

  public static void main (String[] args) {
    String orgDir = "/home/neutron/Rap/lyrics/orglyrics/";
    File orgPath = new File(orgDir);

    if (orgPath.exists()) {

        Collection knowledgegraph = new Collection();
        knowledgegraph.processDir(orgPath);

        for (Document d : knowledgegraph.documents) {
          System.out.println(d);
        }

    } else {

        System.out.println("Org dir not valid");

    }
  }

}

File: src/main/scala/knowledgegraph/KnowledgeGraph.scala
package knowledgegraph

import java.io.File

import collection._

object KnowledgeGraph {
  def main (args: Array[String]): Unit = {
    val orgDir = new File("/Users/davidlarsen/Git/Describe/describe")

    if (orgDir.exists) {
        val knowledgegraph = new Collection()
        val myfiles = knowledgegraph.processDir(orgDir)
    } else {
        println("Org dir not valid");
    }
  }
}
